### PR TITLE
Selectors tests.

### PR DESCRIPTION
**Pull request by pink-bunny**

### CODE REVIEW

File: state/dashboard/__tests__/selectors.test.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    data: {
+      movies: {
+        1: { id: 1, title: 'First mock title' },
+        2: { id: 2, title: 'Second mock title' }
+      }
+    }
+  };
+
+  it('isDashboardMoviesLoading() returns loading status', () => {
+    expect(isDashboardMoviesLoading(state)).toEqual(loading);
```
Comment (plzen) [inline state/dashboard/__tests__/selectors.test.js]: You can use .toBeFalsy()
https://jestjs.io/docs/en/expect#tobefalsy

File: state/dashboard/__tests__/selectors.test.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      ];
+      expect(dashboardMoviesList(state)).toEqual(expectedResult);
+    });
+
+    it('does not have movies and returns null', () => {
+      const emptyMoviesState = {
+        dashboard: { ids: [] },
+        data: { movies: {} }
+      };
+      expect(dashboardMoviesList(emptyMoviesState)).toEqual(null);
```
Comment (plzen) [inline state/dashboard/__tests__/selectors.test.js]: You can use .toBeNull()
https://jestjs.io/docs/en/expect#tobenull

File: state/movie/__tests__/selectors.test.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    data: {
+      movies: {
+        1: { id: 1, title: 'First mock title' },
+        2: { id: 2, title: 'Second mock title' }
+      }
+    }
+  };
+
+  it('isMovieLoading() returns loading status', () => {
+    expect(isMovieLoading(state)).toEqual(loading);
```
Comment (plzen) [inline state/movie/__tests__/selectors.test.js]: You can use .toBeFalsy()
https://jestjs.io/docs/en/expect#tobefalsy

File: state/movie/__tests__/selectors.test.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      const expectedResult = { id: 1, title: 'First mock title' };
+      expect(movieDetails(state)).toEqual(expectedResult);
+    });
+
+    it('does not have movie id and returns null', () => {
+      const emptyMovieState = {
+        movie: { id: null },
+        data: { movies: {} }
+      };
+      expect(movieDetails(emptyMovieState)).toEqual(null);
```
Comment (plzen) [inline state/movie/__tests__/selectors.test.js]: You can use .toBeNull()
https://jestjs.io/docs/en/expect#tobenull

File: state/my_lists/__tests__/selectors.test.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    data: {
+      lists: {
+        1: { id: 1, title: 'First mock title' },
+        2: { id: 2, title: 'Second mock title' }
+      }
+    }
+  };
+
+  it('isMyListsLoading() returns loading status', () => {
+    expect(isMyListsLoading(state)).toEqual(loading);
```
Comment (plzen) [inline state/my_lists/__tests__/selectors.test.js]: You can use .toBeFalsy()
https://jestjs.io/docs/en/expect#tobefalsy

File: state/my_lists/__tests__/selectors.test.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+      ];
+      expect(myListsList(state)).toEqual(expectedResult);
+    });
+
+    it('does not have movies and returns null', () => {
+      const emptyListsState = {
+        myLists: { ids: [] },
+        data: { lists: {} }
+      };
+      expect(myListsList(emptyListsState)).toEqual(null);
```
Comment (plzen) [inline state/my_lists/__tests__/selectors.test.js]: You can use .toBeNull()
https://jestjs.io/docs/en/expect#tobenull

File: state/session/__tests__/selectors.test.js [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+      username
+    }
+  };
+
+  it('getSessionId() returns sesion id', () => {
+    expect(getSessionId(state)).toEqual(sessionId);
+  });
+
+  it('isUserLoggedIn() converts sessionId to bool type and returns it', () => {
+    expect(isUserLoggedIn(state)).toEqual(sessionIdBool);
```
Comment (plzen) [inline state/session/__tests__/selectors.test.js]: You can use .toBeTruthy()
https://jestjs.io/docs/en/expect#tobetruthy

File: state/dashboard/__tests__/selectors.test.js (hunk 1 above)
Comment (pink-bunny) [reply state/dashboard/__tests__/selectors.test.js]: Fixed

File: state/dashboard/__tests__/selectors.test.js (hunk 2 above)
Comment (pink-bunny) [reply state/dashboard/__tests__/selectors.test.js]: Fixed

File: state/movie/__tests__/selectors.test.js (hunk 3 above)
Comment (pink-bunny) [reply state/movie/__tests__/selectors.test.js]: Fixed

File: state/movie/__tests__/selectors.test.js (hunk 4 above)
Comment (pink-bunny) [reply state/movie/__tests__/selectors.test.js]: Fixed

File: state/my_lists/__tests__/selectors.test.js (hunk 5 above)
Comment (pink-bunny) [reply state/my_lists/__tests__/selectors.test.js]: Fixed

File: state/my_lists/__tests__/selectors.test.js (hunk 6 above)
Comment (pink-bunny) [reply state/my_lists/__tests__/selectors.test.js]: Fixed

File: state/session/__tests__/selectors.test.js (hunk 7 above)
Comment (pink-bunny) [reply state/session/__tests__/selectors.test.js]: Fixed

**Pull request merged**